airflow/providers/amazon/aws/sensors/redshift_cluster.py

```diff
@@ -56,8 +56,14 @@ def __init__(
         self.aws_conn_id = aws_conn_id
 
     def poke(self, context: Context):
-        self.log.info("Poking for status : %s\nfor cluster %s", self.target_status, self.cluster_identifier)
-        return self.hook.cluster_status(self.cluster_identifier) == self.target_status
+        current_status = self.hook.cluster_status(self.cluster_identifier)
+        self.log.info(
+            "Poked cluster %s for status '%s', found status '%s'",
```

Comment:
```suggestion
            "Poked cluster %s for status %r, found status %r",
```

Reply:
I'd be happy to apply the suggestion, but can you explain why this would be better ? Both being strings, the output would be exactly the same, wouldn't it ?

Reply:
Yes. Just merely a suggestion for readability and maybe a little future-proofing with all of those statuses if there is some Enum to be created with a nice repr. Certainly isn't a blocking suggestion to use repr or not.